src/classes/BDCBidOutcome.cls

```diff
@@ -1,4 +1,4 @@
-public class BDCBidOutcome {
+public class BDC_BidOutcome {
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.